go/libkb/loaduser.go

```diff
@@ -304,7 +304,9 @@ func LoadUserFromServer(g *GlobalContext, uid keybase1.UID, body *jsonw.Wrapper)
 		if err != nil {
 			return nil, err
 		}
-		body = res.Body.AtKey("them")
+		if body, err = res.Body.AtKey("them").ToArray(); err != nil {
```

Comment:
What do you mean? `AtKey` returns a `go-json.Wrapper`, which has an `err` element of type `go-json.Error`.